crates/hyfetch/src/presets.rs

```diff
@@ -443,6 +481,82 @@ impl Preset {
                 ColorProfile::from_hex_colors(vec!["#226CB5", "#94E7DD", "#FFFFFF", "#636363"])
             },
 
+            Self::Kenochoric => {
+                ColorProfile::from_hex_colors(vec!["#000000", "#2E1569", "#824DB7", "#C7A1D6"])
+            },
+
+            Self::Veldian => ColorProfile::from_hex_colors(vec![
+                "#D182A8", "#FAF6E0", "#69ACBE", "#5D448F", "#3A113E",
+            ]),
+
+            Self::Solian => ColorProfile::from_hex_colors(vec![
+                "#FFF8ED", "#FFE7A8", "#F1B870", "#A56058", "#46281E",
+            ]),
+
+            Self::Lunian => ColorProfile::from_hex_colors(vec![
+                "#2F0E62", "#6F41B1", "#889FDF", "#7DDFD5", "#D2F2E2",
+            ]),
+
+            Self::Polyam => ColorProfile::from_hex_colors(vec![
+                "#FFFFFF", "#FCBF00", "#009FE3", "#E50051", "#340C46",
+            ]),
+
+            Self::Sapphic => ColorProfile::from_hex_colors(vec![
+                "#FD8BA8", "#FBF2FF", "#C76BC5", "#FDD768", "#C76BC5", "#FBF2FF", "#FD8BA8",
+            ]),
+
+            Self::Androgyne => ColorProfile::from_hex_colors(vec!["#FE007F", "#9832FF", "#00B8E7"]),
+
```

Comment:
FWIW, the Python source has a duplicate "genderflux" entry now with different colours. We'll need to find out what happened and see what to do...

Reply:
My investigation seems to suggest this is the same flag as what we already have. The differences can be explained by inaccurate colour-picking.

@BKasin Let's remove this? Perhaps we could redo the colour-picking for the genderflux flag to make sure we have the most accurate colours? All the places I could find ultimately point back to the same source: https://www.deviantart.com/pride-flags/art/Genderflux-1-543925589

Reply:
Confirmed that all the colours in our existing genderflux flag are as accurate as they can be.

Reply:
I believe the repeat one was included with flags from a project called `hyfetch-extra-legal` that recently upstreamed their extra flags and it didn't get caught during that. The colour difference seems to be from using the same flag from different locations to get the source, so one may have had colours distorted by compression or something. I'm not sure which of the two is technically more accurate.